SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -66,8 +75,24 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 		AnyServerRequirementSet.Add(ZoningAttributeSet);
 		AnyServerOrClientRequirementSet.Add(ZoningAttributeSet);
 		AnyServerOrOwningClientRequirementSet.Add(ZoningAttributeSet);
+
+		check(NetDriver->LoadBalanceStrategy != nullptr);
```

Comment:
I assume we have more verbose errors than this, before this point, if LBS is nullptr?

Reply:
Currently, our netdriver logic makes it impossible for you to enable the load balancer without something being assigned.

Reply:
The NetDriver initialization should set and check it, and that should always be called before entity creation, so this should be fine to be a check.